base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -746,8 +830,10 @@ where T: BlockchainBackend
                 .height -
                 1;
             self.reorganize_chain(fork_height, reorg_chain)?;
+            info!(target: LOG_TARGET, "Chain reorg happened");
             return Ok(BlockAddResult::ChainReorg);
         }
+        warn!(target: LOG_TARGET, "Orphan block received");
```

Comment:
The caller will log this level of detail. If think what we want here is:
```
debug!(target: LOG_TARGET, "Orphan block received: {}", block);
```
(note this is debug. Caller will be warn / info)